games-fps/eduke32/eduke32-20181010.7067.ebuild

```diff
@@ -0,0 +1,247 @@
+		unzip -q "${DISTDIR}"/${MY_PN_XXX}-${MY_PV_XXX}.zip xxx_readme.txt || die
+	fi
+	if use opl; then
+		unzip -q "${DISTDIR}"/${MY_PN_OPL}-${MY_PV_OPL}.zip readme.txt || die
+	fi
+	if use sc-55; then
+		unzip -q "${DISTDIR}"/${MY_PN_SC55}-${MY_PV_SC55}.zip readme/music_readme.txt || die
+	fi
+	if use voxels; then
+		unzip -q "${DISTDIR}"/${MY_PN_VOXELS}-${MY_PV_VOXELS}.zip voxelpack_readme.txt || die
+	fi
+}
+
+src_compile() {
+	local myemakeopts=(
+		ALLOCACHE_AS_MALLOC=0
+		AS=$(tc-getAS)
+		CC=$(tc-getCC)
+		CXX=$(tc-getCXX)
+		CLANG=0
+		CPLUSPLUS=1
+		CUSTOMOPT=""
+		DEBUGANYWAY=0
+		F_JUMP_TABLES=""
+		FORCEDEBUG=0
+		HAVE_FLAC=$(usex flac 1 0)
+		HAVE_GTK2=$(usex gtk 1 0)
+		HAVE_VORBIS=$(usex vorbis 1 0)
+		HAVE_XMP=$(usex xmp 1 0)
+		LINKED_GTK=$(usex gtk 1 0)
+		LTO=1
+		LUNATIC=0
+		KRANDDEBUG=0
+		MEMMAP=0
+		MIXERTYPE=SDL
+		NETCODE=$(usex server 1 0)
+		NOASM=0
+		OPTLEVEL=0
+		OPTOPT=""
+		PACKAGE_REPOSITORY=1
+		POLYMER=$(usex opengl 1 0)
+		PRETTY_OUTPUT=0
+		PROFILER=0
+		RELEASE=1
+		RENDERTYPE=SDL
+		SDL_TARGET=2
+		SIMPLE_MENU=0
+		STRIP=""
+		TANDALONE=0
+		STARTUP_WINDOW=$(usex gtk 1 0)
+		USE_OPENGL=$(usex opengl 1 0)
+		USE_LIBVPX=$(usex vpx 1 0)
+		USE_LIBPNG=$(usex png 1 0)
+		USE_LUAJIT_2_1=0
+		WITHOUT_GTK=$(usex gtk 0 1)
+	)
+
+	emake "${myemakeopts[@]}"
+
+	if use tools; then
+		emake utils "${myemakeopts[@]}"
+	fi
+}
+
+src_install() {
+	local binary
+	local binaries=(
+		eduke32
+		mapster32
+		"${FILESDIR}"/eduke32-bin
+	)
+	for binary in "${binaries[@]}"; do
+		dobin "${binary}"
+	done
+
+	if use tools; then
+		local tool
+		local tools=(
+			arttool
+			bsuite
+			cacheinfo
+			generateicon
+			givedepth
+			ivfrate
+			kextract
+			kgroup
+			kmd2tool
+			makesdlkeytrans
+			map2stl
+			md2tool
+			mkpalette
+			transpal
+			unpackssi
+			wad2art
+			wad2map
+		)
+		for tool in "${tools[@]}"; do
+			dobin ${tool}
+		done
+	fi
+
+	keepdir /usr/share/games/eduke32
+	insinto /usr/share/games/eduke32
+	if use hrp; then
+		doins "${DISTDIR}"/${MY_PN_HRP}-${MY_PV_HRP}.zip
+	fi
+	if use offensive; then
+		doins "${DISTDIR}"/${MY_PN_XXX}-${MY_PV_XXX}.zip
+	fi
+	if use opl; then
+		doins "${DISTDIR}"/${MY_PN_OPL}-${MY_PV_OPL}.zip
+	fi
+	if use psx; then
+		doins "${DISTDIR}"/${MY_PN_PSX}-${MY_PV_PSX}.zip
+	fi
+	if use sc-55; then
+		doins "${DISTDIR}"/${MY_PN_SC55}-${MY_PV_SC55}.zip
+	fi
+	if use sdk; then
+		doins -r package/sdk
+	fi
+	if use voxels; then
+		doins "${DISTDIR}"/${MY_PN_VOXELS}-${MY_PV_VOXELS}.zip
+	fi
+
+	newicon "${DISTDIR}"/eduke32_classic.png eduke32.png
+
+	make_desktop_entry eduke32-bin EDuke32 eduke32 Game
+	make_desktop_entry mapster32 Mapster32 eduke32 Game
+
+	local DOCS=( package/sdk/samples/*.txt source/build/doc/*.txt source/duke3d/src/lunatic/doc/*.txt )
+	if use hrp; then
+		DOCS+=( "${WORKDIR}"/hrp_readme.txt "${WORKDIR}"/hrp_todo.txt )
+	fi
+	if use offensive; then
+		DOCS+=( "${WORKDIR}"/xxx_readme.txt )
+	fi
+	if use opl; then
+		DOCS+=( "${WORKDIR}"/readme.txt )
```

Comment:
Isn't this a bad name for subcomponent readme?

Reply:
Yes, you are right. I've renamed it now to `opl_readme.txt`, to make it more consistent.